### PR TITLE
migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by lufia**
I've migrate both Travis CI and AppVeyor tasks into a GitHub Actions workflow.

Note: The Windows runner can run many POSIX commands. IMO, these commands seems to convert `\n` to `\r\n` on writing text files to the disk, to convert `\r\n` to `\n` on reading text files. However, because Go program don't convert them some tests were failed. Therefore I decieed to skip them temporaritly.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    steps:
+    - run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+          rpm devscripts debhelper fakeroot \
+          crossbuild-essential-arm64 crossbuild-essential-armhf
+        mkdir -p ~/rpmbuild/{BUILD,BUILDROOT,RPMS,SOURCES,SPECS,SRPMS}
+    - uses: actions/setup-go@v2
```
Comment (astj) [inline .github/workflows/ci.yml:58]: Should we also use `actions/cache@v2` for `~/go/pkg/mod` here?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      with:
+        go-version: 1.14.x
+    - uses: actions/checkout@v2
+    - run: |
+        go get github.com/x-motemen/gobump/cmd/gobump
+        go get github.com/mackerelio/golib/cmd/mackerel-github-release
+        make all
+        mackerel-github-release && mackerel-github-release -staging
+      env:
+        GITHUB_USER: mackerelbot
```
Comment (astj) [inline .github/workflows/ci.yml:68]: Is GITHUB_USER necessary?
in mackerel-agent, we used only GITHUB_TOKEN https://github.com/mackerelio/mackerel-agent/blob/master/.github/workflows/go.yml#L87

**Review by astj — APPROVED** [review]
Seems good, but added some comments

File: .github/workflows/ci.yml (hunk 1 above)
Comment (lufia) [reply .github/workflows/ci.yml:58]: I overlooked that; will add it soon.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (lufia) [reply .github/workflows/ci.yml:68]: I see...

**Review by astj — APPROVED** [review]
lgtm!

**Pull request merged**